pkg/app/app.go

```diff
@@ -21,6 +21,7 @@ import (
 	"os"
 	"os/signal"
 	"path/filepath"
+	"sigs.k8s.io/kubetest2/pkg/artifacts"
```

Comment:
non-blocking nit: typically in Kubernetes we group imports like this:

stdlib

third party packages

packages from the same project